book/types_of_data.md

```diff
@@ -22,7 +22,7 @@ The [`describe`](/commands/docs/describe.md) command returns the type of a data
 | Booleans          | `true`                                                                |
 | Dates             | `2000-01-01`                                                          |
 | Durations         | `2min + 12sec`                                                        |
-| File sizes        | `64mb`                                                                |
+| File sizes        | `64MB`                                                                |
```

Comment:
Maybe this should be `64mb` or `64MB` to drive home the fact that nushell doesn't care about case in these instances.

Reply:
```suggestion
| File sizes        | `64MB`, `64Mb`, `64mb`                                                |
```

Too many? I put a couple that are equivalent

Reply:
I'm fine with it